bin/yaml/libraries.yaml

```diff
@@ -659,6 +659,40 @@ libraries:
       - 6.1.2
       type: s3tarballs
       untar_dir: hip-amd-rocm-{{name}}
+    hpx:
+      after_stage_script:
+      - mkdir build
```

Comment:
most of this is not needed, you only have to set `extra_cmake_arg` for the -D parameters, set built_type to cmake and set `package_install: true`

for an example, see https://github.com/compiler-explorer/infra/blob/d95c2e9b6ea1ab7a6475052632a70d703224204f/bin/yaml/libraries.yaml#L302

Reply:
Hello @partouf Thanks for the review. I was initially planning to follow similar approach to `cpptrace`. But, I am unable to reproduce it locally. I run the command `./bin/ce_install install libraries` and I can only see `cpptrace` versions cloning but not actually building anything.

Reply:
You dont really need to, but theres some pointers here https://github.com/compiler-explorer/compiler-explorer/blob/main/docs/internal/AddingLinkableLibrary.md

Reply:
Thanks a lot @partouf, that really simplied things.